### PR TITLE
Remove user MFA credentials endpoint and command.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: internal/auth/auth.go [hunk 1]
```diff
@@ -40,38 +40,41 @@ const (
...
+	ActionCreateParentAgent        = "createParentAgent"
+	ActionReadAdmin                = "readAdmin"
+	ActionWriteAdmin               = "writeAdmin"
+	ActionReadGroups               = "readGroups"
+	ActionWriteGroups              = "writeGroups"
+	ActionReadSSHKeys              = "readSSHKeys"
+	ActionWriteSSHKeys             = "writeSSHKeys"
+	ActionLogin                    = "login"
+	ActionReadDischargeToken       = "read-discharge-token"
+	ActionRemoveUserMFACredentials = "remove-user-mfa-credentials"
```
Comment (mhilton) [inline internal/auth/auth.go]: I know "read-discharge-token" is formatted like this, but I'd prefer if we had a more consistent format. I think "removeUserMFACredentials"

File: internal/v1/users.go [hunk 2]
```diff
@@ -92,6 +92,23 @@ func (h *handler) QueryUsers(p httprequest.Params, r *params.QueryUsersRequest)
 	return usernames, nil
 }
 
+// RemoveUserMFACredentials removes all MFA credentials for a user.
+func (h *handler) RemoveUserMFACredentials(p httprequest.Params, r *params.RemoveUserMFACredentialsRequest) error {
```
Comment (mhilton) [inline internal/v1/users.go]: Would ClearUserMFACredentials be a better name?

**Review by mhilton — APPROVED** [review]
LGTM with a couple of suggestions

File: internal/auth/auth.go (hunk 1 above)
Comment (alesstimec) [reply internal/auth/auth.go]: done

File: internal/v1/users.go (hunk 2 above)
Comment (alesstimec) [reply internal/v1/users.go]: done

**Pull request merged**